test/jdk/javax/net/ssl/templates/SSLSocketSSLEngineTemplate.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2011, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Can you please check test `sun/security/ssl/SSLEngineImpl/SSLEngineBadBufferArrayAccess.java` as this template test was a copy of it? Suggest you to check if this template copy can be removed now as we have other template classes for SSLSocket/SSLEngine client and server.

Reply:
Removed SSLSocketSSLEngine.java